rules/macos/credential_access_suspicious_web_browser_sensitive_file_access.toml

```diff
@@ -52,24 +52,15 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "macos" and event.type in ("start", "process_started") and
-  process.args :
-    (
-      "/Users/*/Library/Application Support/Google/Chrome/Default/Login Data",
-      "/Users/*/Library/Application Support/Google/Chrome/Default/Cookies",
-      "/Users/*/Library/Application Support/Google/Chrome/Profile*/Cookies",
-      "/Users/*/Library/Cookies*",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/cookies.sqlite",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/key*.db",
-      "/Users/*/Library/Application Support/Firefox/Profiles/*.default/logins.json",
-      "Login Data",
-      "Cookies.binarycookies",
-      "key4.db",
-      "key3.db",
-      "logins.json",
-      "cookies.sqlite"
-    ) and 
-  not (process.name : "wordexp-helper" and process.parent.name : ("elastic-agent", "elastic-endpoint"))
+file where event.action == "open" and process.executable != null and 
+ file.name : ("cookies.sqlite", 
```

Comment:
We may be able to specify additional sensitive files as well as extensions for certain files, like DB files. Here is a potential list:

```
_**File Extensions**_

SQLite Database Files: Used by browsers to store cookies, session data, and history.
.sqlite (e.g., cookies.sqlite, places.sqlite)

JSON Files: Often used for storing browser settings, bookmarks, and extensions information.
.json (e.g., bookmarks.json, logins.json)

Binary Cookies Files: Specifically used by Safari to store cookies.
.binarycookies (e.g., Cookies.binarycookies)

Web Data Files: Chrome and derivatives use these for storing autofill form data and other web-related data.
.db (e.g., Login Data, Web Data)

Plist Files: Used by Safari for bookmarks and preferences.
.plist (e.g., Bookmarks.plist)

**_Specific File Names_**

Chrome, Chromium, and Edge:

- Cookies
- Login Data
- Web Data
- History
- Bookmarks

Firefox:

- cookies.sqlite
- logins.json
- key4.db
- places.sqlite

Safari:

- Cookies.binarycookies
- History.db
- Bookmarks.plist

```

I think Chrome is a little bit harder for file names specifically as they are so generic and typically bundled.

Reply:
The file names listed are all the web browsers files we collect for Chrome, Firefox, and Edge.